crates/tauri-bundler/src/bundle/windows/nsis/mod.rs

```diff
@@ -51,6 +51,14 @@ const NSIS_REQUIRED_FILES: &[&str] = &[
   "Include/nsDialogs.nsh",
   "Include/WinMessages.nsh",
 ];
+#[cfg(target_os = "windows")]
+const NSIS_PLUGIN_DLLS: &[&str] = &[
+  "NSISdl.dll",
+  "StartMenu.dll",
+  "System.dll",
+  "nsDialogs.dll",
+  "nsis_tauri_utils.dll",
```

Comment:
I suspect this one to be though, maybe we should only sign this

Reply:
I want to sign them all just for any case. Maybe we should add option to enable signNsisPlugins?

Reply:
I remember that AVs flagged the plugins DLLs as malware just because they are bot signed

Reply:
let's try with signing all then.